src/Tokenizer/Tokens.php

```diff
@@ -593,39 +594,50 @@ public function getPrevNonWhitespace($index, $whitespaces = null)
      * Get index for closest previous token of given kind.
      * This method is shorthand for getTokenOfKindSibling method.
      *
-     * @param int   $index         token index
-     * @param array $tokens        possible tokens
-     * @param bool  $caseSensitive perform a case sensitive comparison
+     * @param int      $index         token index
+     * @param array    $tokens        possible tokens
+     * @param bool     $caseSensitive perform a case sensitive comparison
+     * @param int|null $limit         false no limit
      *
      * @return int|null
      */
-    public function getPrevTokenOfKind($index, array $tokens = array(), $caseSensitive = true)
+    public function getPrevTokenOfKind($index, array $tokens = array(), $caseSensitive = true, $limit = null)
     {
-        return $this->getTokenOfKindSibling($index, -1, $tokens, $caseSensitive);
+        return $this->getTokenOfKindSibling($index, -1, $limit, $tokens, $caseSensitive);
```

Comment:
also, having different order of params for general `getTokenOfKindSibling` method and it's aliases is not intuitive